WordPress/AbstractFunctionRestrictionsSniff.php

```diff
@@ -229,7 +281,7 @@ public function is_targetted_token( $stackPtr ) {
 	 *                  normal file processing.
 	 */
 	public function check_for_matches( $stackPtr ) {
-		$token_content = strtolower( $this->tokens[ $stackPtr ]['content'] );
+		$token_content = strtolower( $this->strip_quotes( $this->tokens[ $stackPtr ]['content'] ) );
```

Comment:
Please add a comment here about the use of `strip_quotes()` as it is not needed for the "normal" function check.